File: powtórzenie klasa 2/bmi.cpp
/*
 * bmi.cpp

 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    float waga, wzrost, bmi;
    waga = wzrost = bmi = 0;
	cout<< "podaj swoją wagę"<<endl;
    cin >> waga;
    cout<< "podaj swój wzrost"<<endl;
    cin >> wzrost;
    bmi = waga/(wzrost*wzrost);
    cout <<  "Bmi:" << bmi;
    
    if(bmi>=30)
    {
        cout << "otyłość";
    }
    if(bmi>=25)
    {
        cout << "nadwaga";
    }
    if(bmi<18.5)
    {
        cout << "niedowaga";
    }
    if(bmi >18.5 && bmi<24.9)
    {
        cout<< "norma";    
    }
    
	return 0;
}

File: powtórzenie klasa 2/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    waga = float(input('Podaj wage: '))
    print (waga)
    wzrost = float(input('podaj wzrost: '))
    print (wzrost)
    bmi = float(waga/(wzrost * wzrost))

    print ( bmi )


    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
